src/Post_Type.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Post_Type class file.
+ *
+ * @package  studiometa/wp-factories
+ */
+
+namespace Studiometa\WP\Factories;
+
+/**
+ * Post_Type Class
+ */
+class Post_Type {
+	/**
+	 * Generate labels
+	 *
+	 * @param  string $singular           Singular name.
+	 * @param  string $plural             Plural name.
+	 * @param  string $translation_domain Translation domain name.
+	 * @return array                      Labels
+	 */
+	public static function generate_labels( string $singular, string $plural, string $translation_domain = null ) {
+		$singular_lowercase = strtolower( $singular );
```

Comment:
Maybe the `$singular_lowercase` and `$plural_lowercase` values should be passed as arguments and fallback to the `strtolower` value. It will allow us to specify custom values for edge cases.